### ISSUE TITLE
(Feature Request) stratum info

### ISSUE THREAD

**Issue by Titaniumtown** [bedrocklinux/bedrocklinux-userland#173]
There should be (if there isn't already) a command that returns info about an installed stratum.

What I think such a command should be like:
```
brl info fedora

Operating system: Fedora
Version: 32
Architecture: x86_64
Space taken up on disk: 10 gb
```

**Comment by paradigm** [comment]
Bedrock has survived as long as it has by keeping the project scope something the very limited available developer/maintainer hours can do.  In general, Bedrock is about making features from other projects accessible and _not_ about reinventing the wheel where other projects have already done something.  Here, it seems like most of what you're looking for is provided by other projects such as [neofetch](https://github.com/dylanaraps/neofetch):

```
$ strat -r fedora neofetch distro
distro: Fedora 32 (Thirty Two) x86_64
```

I don't know off the top of my head a good way to get disk usage of a stratum, which makes implementing your last field difficult.  For specifically disabled strata which are not their own partition, you can query disk usage with `du`:

```
$ sudo brl disable fedora
$ sudo du /bedrock/strata/fedora
955M    /bedrock/strata/fedora
```

Moreover, if Bedrock were to create its own implementation despite the fact it's largely reinventing the wheel, this is going to require maintenance work to follow upstream distro changes, and we're badly hurting for distro maintainer hours as it is.

That having been said, I could be missing something.  Do you see a use case for this feature sufficient to overcome the above reasons not to implement it?

**Comment by Titaniumtown** [comment]
I guess I could write something once #175 is merged

**Comment by paradigm** [comment]
To be clear, the reasons I listed against are _not_ about the one-time cost of an initial implementation.  Even if you write it, if you don't provide a reason for why it should overcome the reasons listed above, it's unlikely to be merged.

If your considering writing something for your personal needs with no interest in upstreaming it, there's no need to wait for #175; you can write a stand-alone utility to do this and drop it in whenever #175 is ready.
